portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```
@@ -0,0 +1,101 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that processes text. */
+@WebServlet("/comments")
+public final class CommentsServlet extends HttpServlet {
+
+  private ArrayList<Comment> theComments = new ArrayList<Comment>(); 
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json;");
+    response.getWriter().println(new Gson().toJson(theComments));
+    }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    String name = getParameter(request,"name-input", "");
+    String payload = getParameter(request,"comment-input", "");
```

Consider adding validation here. For example, instead of just defaulting to an empty name, empty comment, or star rating of 0, you could return an error. Mostly depends on whether you want to allow "empty" values.